### ISSUE TITLE
Have example cleanup scripts clean up directory paths.

### ISSUE THREAD

**Issue by xenophenes** [CrunchyData/crunchy-containers#557]
**I'm submitting a ...**

 
 - [x] bug report

**What is the current behavior?**

The only example that currently cleans up the directories is **pitr**. The rest leave the directories as-is which can cause issues with HostPath and NFS storage systems.

**What is the motivation or use case for changing the behavior?**

Consistency and usability of the examples. The cleanup scripts should clean up the directories where it makes sense to when running the cleanup.sh script. Offhand, the only examples exempt from this would be the backup examples.

**Comment by xenophenes** [comment]
@jmccormick2001 @jasonodonnell Any specific examples where the directory paths should be left after cleanup? The only scenarios I can think of would be the backup examples (pgdump, pgbackrest, backup).

**Comment by jasonodonnell** [comment]
@xenophenes I can't think of anything else, it's mostly related to backups.

**Comment by jasonodonnell** [comment]
@xenophenes something else I would love to see here that would really simplify a lot of the cleanup operations we do.  I suggest we move to cleaning up objects created in our examples via labels instead of the object name.  This will greatly simplify a lot of cleanup operations.

For example today, take the statefulset cleanup script, we have to keep track of all the objects names and delete them like this:

```bash
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} statefulset statefulset
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} sa statefulset-sa
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} clusterrolebinding statefulset-sa
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} pvc statefulset-pgdata
if [ -z "$CCP_STORAGE_CLASS" ]; then
  ${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} pv statefulset-pgdata
fi
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} service statefulset statefulset-primary statefulset-replica
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} pod statefulset-0 statefulset-1
```

Via labels this can be reduced to simply (provided we attach metadata labels to each of the objects being created):

```bash
${CCP_CLI?} delete --namespace=${CCP_NAMESPACE?} \
    clusterrolebinding,deployment,pod,pv,pvc,sa,service,replicaset,statefulset  \
    -l 'example=statefulset'
```

**Comment by xenophenes** [comment]
I absolutely agree, but I feel as though it should be created and treated as a separate issue. More than happy to address it and have it assigned to me.

**Comment by cahoonpwork** [comment]
This has been merged into 2.0.0 and a new issue created to perform this action by labels
